nflgame/update_players.py

```diff
@@ -181,6 +193,7 @@ def meta_from_soup_row(team, soup_row):
     return {
         'team': team,
         'profile_id': profile_id_from_url(profile_url),
+        'esb_id': esb,
```

Comment:
I don't think this is sufficient. I think you also need to add it as a member on the `Player` object?

Do you have code that works with this patch?